### PR TITLE
Command: refactor to remove httpErrorHandler global variable

### LINKED ISSUE #2016: Command: refactor to remove httpErrorHandler global variable
Refactor the above-identified service to remove the above-identified global variable (which is defined in the service's init.go).

This work has already been completed in the system management agent.  Use it as a reference.  See https://github.com/edgexfoundry/edgex-go/blob/master/internal/system/agent/router.go and https://github.com/edgexfoundry/edgex-go/blob/master/internal/system/agent/init.go.

You will need to update route creation to explicitly pass the individual required dependencies to the controller functions.  This will bubble down through the implementation; that is, you'll need to pass those dependencies down the call chain to where they're currently referencing the global variable.  Do not pass the dependency injection container into any controller functions.

You can also create any long-lived dependencies in the service's bootstrap and use the dependency injection container to hold them.  This was done in the system management agent; look there for an example.

Add me as the sole reviewer of your PR.

Note this is Geneva work and should not be merged into master prior to Fuji branching.  As there are a series of these issues for this service (each refactoring away a global variable), assign all of this service's related issues to you and use the branch created for this PR as the starting point for your next PR to avoid extensive conflicts. 

Depends on https://github.com/edgexfoundry/edgex-go/issues/2015

### PR DESCRIPTION
**Pull request by brandonforster**
Fixes #2016.

Must be merged AFTER #2079.

### CODE REVIEW

**Comment by brandonforster** [comment]
This is trivial because core-command doesn't actually use errorConcepts. I added the global here in prep for the second half of #1860 which has now been superseded by #2012. I expect that #2012 will implement actual direct injection for the httpErrorHandler.

File: internal/core/command/init.go [hunk 1]
```diff
@@ -34,14 +34,11 @@ import (
...
 
-// Global ErrorConcept variables
-var httpErrorHandler errorconcept.ErrorHandler
-
 // BootstrapHandler fulfills the BootstrapHandler contract and performs initialization needed by the command service.
 func BootstrapHandler(wg *sync.WaitGroup, ctx context.Context, startupTimer startup.Timer, dic *di.Container) bool {
 	loggingClient := bootstrapContainer.LoggingClientFrom(dic.Get)
 	registryClient := bootstrapContainer.RegistryFrom(dic.Get)
-	httpErrorHandler = errorconcept.NewErrorHandler(loggingClient)
+	errorconcept.NewErrorHandler(loggingClient)
```
Comment (michaelestrin) [inline internal/core/command/init.go]: Is this call actually needed?

Comment (brandonforster) [reply internal/core/command/init.go]: Not really, but without it we remove the logging client pull which is then going to get put right back by #2012.

Truthfully, I think we should close this PR without merge, close the issue, and handle this particular variable as a part of #2012. Anything reasonable done here is going to be immediately undone by #2012, and anything silly (like the current status of the PR) probably isn't worth doing.

**Comment by michaelestrin** [comment]
> Truthfully, I think we should close this PR without merge, close the issue, and handle this particular variable as a part of #2012. Anything reasonable done here is going to be immediately undone by #2012, and anything silly (like the current status of the PR) probably isn't worth doing.

I'm aligned with that.  Will close this PR and the corresponding issue.

**Pull request closed without merging**